.github/workflows/CD-publish-dev-container.yml

```diff
@@ -0,0 +1,61 @@
+name: Publish Dev Container Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      build_secret:
+        type: string
+        description: Build secret
+
+jobs:
+  check-user-permissions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check user permission
+        uses: stjude/proteinpaint/.github/actions/check-user-permissions@master
+        with:
+          BUILD_SECRET: ${{ secrets.BUILD_SECRET }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          INPUT_BUILD_SECRET: ${{ github.event.inputs.build_secret }}
+
+  build:
+    needs: check-user-permissions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.PAT }}
+
+      - name: Login to GitHub Container Registry
+        run: |
+          echo $CR_PAT | docker login ghcr.io -u stjude --password-stdin
+        env:
+          CR_PAT: ${{ secrets.PAT }}
+
+      - name: Login to Github
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.PAT }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
What is `QEMU` for? To be able to run x86/intel-built image in ARM machines?

Reply:
Related to my question above: If it's possible to use a native arm64 build, it would be preferable than using QEMU, as mentioned in https://github.blog/news-insights/product-news/arm64-on-github-actions-powering-faster-more-efficient-build-systems/. So, a user would need to use the deps/base image that's appropriate to their machine CPU  architecture. Not sure if this option has already been explored/researched for our dev use case.

Reply:
You are right, it works without QEMU. At one point it wasn't working, since I got an error that was ARM-related, but now it has worked out. I've built the images again.